scripts/actions/add-files-to-translation-queue.js

```diff
@@ -35,6 +32,29 @@ const slugIntersection = (pendingFiles, filesToRemove) =>
 const humanTranslatedProjectID = process.env.HUMAN_TRANSLATION_PROJECT_ID;
 const machineTranslatedProjectID = process.env.MACHINE_TRANSLATION_PROJECT_ID;
 
+/**
+ * Loads the Exclusions yaml file and converts it to a JSON object.
+ * @returns {Object} The Exclusions yaml file as a JSON object.
+ */
+const getExclusions = () => {
+  return yaml.load(fs.readFileSync(path.join(process.cwd(), EXCLUSIONS_FILE)));
+};
+
+/**
+ * Determines if a file should be included based on data from an exclusions file
+ * @param {Object[]} fileData The files to check
+ * @param {Object[]} exclusions The exclusions file
+ * @returns {Object[]} The files that should be included
+ */
+const excludeFiles = (fileData, exclusions) => {
+  return fileData.filter(
+    ({ filename, locale, fileType }) =>
+      !exclusions.excludePath[locale]?.some((exclus) =>
```

Comment:
i would prefer a better name to exclus. usually the temp name of variables for anonymous functions is not something i nitpick, but this left me scratching my head a bit 🙈 

maybe something like `excludedPath`, `excludedDir` or something similar.

Reply:
It's actually latin for exclude so 🤷 😂 excellent point